website/content/docs/upgrading/index.mdx

```diff
@@ -18,6 +18,8 @@ keep in mind when using Consul.
 
 This page documents how to upgrade Consul when a new version is released.
 
+~> For Consul Enterprise, see the [Automated Upgrades documentation](/docs/enterprise/upgrades).
```

Comment:
Is there a reason why you chose the yellow callout instead of `-> **Tip:**`?

Reply:
Nope, copied from an old PR way back so prob didn't know the colors back then :-) just tweaked